spring-boot-project/spring-boot/src/test/java/org/springframework/boot/logging/logback/LogbackLoggingSystemTests.java

```diff
@@ -93,8 +94,9 @@
  * @author Scott Frederick
  * @author Jonatan Ivanov
  * @author Moritz Halbritter
+ * @author Mark Chesney
  */
-@ExtendWith(OutputCaptureExtension.class)
+@ExtendWith({ MockitoExtension.class, OutputCaptureExtension.class })
```

Comment:
Is this change accidental? It doesn't seem related to the other changes?

Reply:
This appeared in #41885, which was reverted.

Without this, I ran into [this issue](https://github.com/mockito/mockito/issues/1379#issuecomment-1967392526) in my environment, which failed the tests that use `Mockito.mock` within this class for being unable to delete `mockitoboot*.jar` from the JUnit temp directory. 

With the change, the test is less finicky on Windows.

That said, I can take it out if required.

Reply:
I don't think it's needed. Regardless, it isn't related to this change so I think it should be taken out. https://github.com/spring-projects/spring-boot/issues/41905 has hopefully helped here.